src/main/java/frc/robot/RobotContainer.java

```diff
@@ -131,27 +131,21 @@ public RobotContainer() {
 
   private void configureAutoBindings() {
     NamedCommands.registerCommand("PrepPlace",
-        Commands.sequence(
-            Commands.deferredProxy(
-                () -> subStateMachine.tryState(RobotState.PREP_CORAL_L3))));
+        Commands.sequence(TRY_PREP_CORAL_L3));
```

Comment:
Looks like when we are putting our TRY_STATE commands in a `Commands.sequence`, we have to do `.asProxy()`. Do this similar thing for all of them
```suggestion
        Commands.sequence(TRY_PREP_CORAL_L3).asProxy());
```